server/db/043-add-number-of-alerts-to-sessions.sql

```diff
@@ -0,0 +1,37 @@
+DO $migration$
```

Comment:
Everything in here looks OK to me. Nothing that would obviously not make the migration fail. When did you run it on Dev? It looks like the 43rd migration was run on Mon, Dec 11:
```
sensors_api_development=> select * from migrations where id=43;
 id |          created_at           
----+-------------------------------
 43 | 2023-12-11 23:09:27.944375+00
(1 row)
```

According to the GitHub Actions (https://github.com/bravetechnologycoop/BraveSensor/actions/runs/7174487582), that was this commit (`e6e1b26f7218b3d499179ce234c6d803cab312ee`) on your branch (`mdb-add-number-of-alerts-to-sessions`).

Which had this code in it: https://github.com/bravetechnologycoop/BraveSensor/blob/e6e1b26f7218b3d499179ce234c6d803cab312ee/server/db/043-add-number-of-alerts-to-sessions.sql

So that would be why your latest version of migration 43 didn't run. If you do make changes and want to re-run the script in Dev. You could reverse your changes, remove the row from the `migrations` table that has `id=43` and then it would run again.

Reply:
Haha, that makes sense! OK, I'll remove that row from the migrations table (and the column for number of alerts) to triple check it works as expected.

Reply:
And remove the `RESET` value from the `enum` too :-)